app/controllers/admin/users_controller.rb

```diff
@@ -34,9 +34,9 @@ def update
     @user = User.find(params[:id])
```

Comment:
And again. Although the logic is sound maybe it is better somewhere else rather than the controller?

Reply:
Make sense